src/components/Dashboard/Overview.tsx

```diff
@@ -310,7 +309,7 @@ function Overview({ settings, getMarketHistory }: OverviewProps) {
           </div>
           {/* <p className="value">{`$${
               (settings.marketType || 'supply') === 'supply'
-                ? new BigNumber(marketInfo.totalSupply || 0).div(new BigNumber(10).pow(vtokenDecimals)).dp(2, 1).toString(10)
+                ? new BigNumber(marketInfo.totalSupply || 0).div(new BigNumber(10).pow(VTOKEN_DECIMALS)).dp(2, 1).toString(10)
```

Comment:
Do we need to maintain commented code? Should we delete it?

Reply:
No, good point. I did a global search and replace and didn't see this affected code that was commented out. I'll remove it!